tests/unit/StandaloneOfficeAddressFiling.spec.ts

```diff
@@ -204,7 +210,7 @@ describe('Standalone Office Address Filing - Part 1 - UI', () => {
       certifyFormValid: false,
       addressesFormValid: false
     })
-    await vm.$store.commit('filingData', []) // no data
+    await rootStore.setFilingData([]) // no data
```

Comment:
This doesn't need to be awaited. Ditto all the other calls to this in the unit tests.

Reply:
Some of the tests are failing without the await which is so weird since I thought exactly the same thing. I'm not really sure why.

For example in this file, 3 tests are failing without the await.

Reply:
If it works with an `await Vue.nextTick()` after line 213 then I prefer that instead -- it suggests we need to wait for something to happen in the component, not that setFilingData() is async.

Reply:
I tested this and it works fine without the await in all tests except 3 -- and awaiting nextTick after setFilingData works for those. That's my preferred implementation.

Reply:
looks fixed